docs/proposal/3_5_schedule.tex

```diff
@@ -7,6 +13,10 @@ \subsubsection{Major Schedule}
 
 
 \subsubsection{Weekly Schedule}
+
+TODO: also make a weekly schedule outlining our availablity (or lack of, due
```

Comment:
Keep in mind, that we don't have much prior knowledge of our course loads (especially in 2nd term), so I'm not entirely sure how confident we can be here. Perhaps brief platitudes here.